x-pack/plugins/observability/public/config/paths.ts

```diff
@@ -8,6 +8,7 @@
 export const paths = {
   observability: {
     alerts: '/app/observability/alerts',
+    ruleDetails: (ruleId: string) => `/app/observability/alerts/rules/${encodeURI(ruleId)}`,
```

Comment:
What do you think of using this var for the URL https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/public/pages/rule_details/config.ts#L22?

Reply:
Yep great suggestion! I'll change it